app/models/post.rb

```diff
@@ -1,2 +1,13 @@
 class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  has_many :comments, dependent: :destroy
+  has_many :likes, dependent: :destroy
+
+  def update_counter
+    author.increment!(:posts_counter)
+  end
+
+  def recent_comments
+    comments.order(created_at: :desc).limit(5)
+  end
 end
```

Comment:
- [ ] [OPTIONAL] You can also use one of the rails active call backs: `after_save` to call the `#update_post_counter` method every time a new post has been saved to database, then make the `update_post_counter` method private
```ruby
  after_save :update_post_counter

  private

  def update_post_counter
    author.increment!(:posts_counter)
  end
```

Reply:
Thanks @Whoistolu